### PR TITLE
[Feat] Show mnemonic without a trace

### PR DESCRIPTION
**Pull request by adriantpaez**
Generated mnemonic is shown using the `less` command. The user can copy the mnemonic from the terminal and after that close the `less` buffer with `q`. With this solution, the mnemonic not remains on the terminal history output.

## Changes:
- New command builder `BuildOpenTextEditor`
- Update the `keys` command and its tests

## Types of changes

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: cli/keys.go [hunk 1]
```diff
@@ -164,3 +162,22 @@ func KeysCmd(prompt prompts.Prompt) *cobra.Command {
...
 	return cmd
 }
+
+func saveMnemonic(mnemonic string) error {
+	file, err := ioutil.TempFile(os.TempDir(), "sedge_mnemonic")
+	if err != nil {
+		return err
+	}
+	defer os.Remove(file.Name())
+	if _, err := file.WriteString(fmt.Sprintf("Mnemonic:\n\n%s\n\n", mnemonic)); err != nil {
```
Comment (AntiD2ta) [inline cli/keys.go:172]: It would be great to show here a constant message advising the user to write down the mnemonic and instructions on how to exit the text editor.  A good reference for a message here is what the [staking-deposit-cli](https://github.com/ethereum/staking-deposit-cli/blob/76ed78224fdfe3daca788d12442b3d1a37978296/staking_deposit/intl/en/cli/new_mnemonic.json) tool shows.

File: CHANGELOG.md [hunk 2]
```diff
@@ -35,6 +35,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Add missing `depends_on` tag to lodestar validator.
 - Fix wrong fork version in Gnosis network config.
+- Show mnemonic without a trace (linux, MacOS).
```
Comment (AntiD2ta) [inline CHANGELOG.md:38]: Put this under Unreleased
```suggestion
- When generating new mnemonic, show it without a trace (linux, MacOS).
```

**Review by AntiD2ta — CHANGES_REQUESTED** [review]
Three minor changes requested. Good job

File: cli/keys.go [hunk 3]
```diff
@@ -164,3 +162,22 @@ func KeysCmd(prompt prompts.Prompt) *cobra.Command {
...
+		return fmt.Errorf("error writing mnemonic to tempfile: %v", err)
```
[the remark this answers is not in the document]
Comment (adriantpaez) [reply cli/keys.go]: Done -> https://github.com/NethermindEth/sedge/pull/169/commits/8d61d61116c955b69b2449c9a0555162543920da

File: cli/keys.go (hunk 1 above)
Comment (adriantpaez) [reply cli/keys.go]: Done -> https://github.com/NethermindEth/sedge/pull/169/commits/8d61d61116c955b69b2449c9a0555162543920da

**Review by AntiD2ta — APPROVED** [review]

File: CHANGELOG.md [hunk 4]
```diff
@@ -10,9 +10,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 
 - Fixed validator restart failure:
-  - Validator waits a grace period (2 epochs) before starting.
-  - Validator waits for the consensus sync to finish: verifying that the `/eth/v1/node/health` consensus endpoint returns with the HTTP code 200 in its docker-compose `healthcheck` condition so that the validator waits for the consensus be healthy. This replace the older track sync method.
-  - Validator service now always restarts unless stopped.
+- Validator waits a grace period (2 epochs) before starting.
+- Validator waits for the consensus sync to finish: verifying that the `/eth/v1/node/health` consensus endpoint returns with the HTTP code 200 in its docker-compose `healthcheck` condition so that the validator waits for the consensus be healthy. This replace the older track sync method.
+- Validator service now always restarts unless stopped.
```
Comment (AntiD2ta) [inline CHANGELOG.md:15]: Revert this change

File: CHANGELOG.md [hunk 5]
```diff
@@ -10,9 +10,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+
+### Fixed
+- When generating new mnemonic, show it without a trace (linux, MacOS).
```
Comment (AntiD2ta) [inline CHANGELOG.md:18]:
```suggestion
- When generating new mnemonic, show it without a trace.
```

File: configs/messages.go [hunk 6]
```diff
@@ -115,5 +115,15 @@ Follow https://launchpad.ethereum.org/ and happy staking!`
...
-	VersionUpdated = "You are running the latest version of sedge. Version: "
+	VersionUpdated       = "You are running the latest version of sedge. Version: "
+	MnemonicPresentation = `
+This is your mnemonic:
+
+%s
+
+The mnemonic is the only way to get your deposit back, write it down now and keep it in a safe place.
+
+Press the <q> key when you have written down your mnemonic.
```
Comment (AntiD2ta) [inline configs/messages.go:126]: Should this need to be updated for Windows and `notepad` @cbermudez97 ?

**Review by AntiD2ta — CHANGES_REQUESTED** [review]
Almost there, just a few things to check and fix

File: configs/messages.go (hunk 6 above)
Comment (cbermudez97) [reply configs/messages.go]: Updated

**Review by AntiD2ta — APPROVED** [review]
Approved!
